charts/incubator/adguardhome-sync/values.yaml

```diff
@@ -1,26 +1,41 @@
-env:
-  CONFIGFILE: ""
-  UMASK: ""
 image:
   pullPolicy: IfNotPresent
   repository: tccr.io/truecharts/adguardhome-sync
-  tag: latest@sha256:b180e316545dc3ab12b620a20b3c07b7b40b8dfc121f60fbdbfa045840136bd3
-persistence:
-  config:
-    enabled: true
-    mountPath: /config
-podSecurityContext:
-  runAsGroup: 0
-  runAsUser: 0
-securityContext:
-  readOnlyRootFilesystem: false
-  runAsNonRoot: false
+  tag: 0.4.12@sha256:2470e0383ffffd9208b3149015f8ac383d60acc878585df9d6db6110af2ac43d
+
+args:
+  - run
+
+env:
+  ORIGIN_URL: "https://192.168.1.2:3000"
+  ORIGIN_USERNAME: "username"
+  ORIGIN_PASSWORD: "password"
+  REPLICA_URL: "http://192.168.1.3"
+  REPLICA_USERNAME: "username"
+  REPLICA_PASSWORD: "password"
+  CRON: "*/10 * * * *"
+  RUNONSTART: true
+  # REPLICA_AUTOSETUP: true # if true, AdGuardHome is automatically initialized.
+  # REPLICA_INTERFACENAME: ens18 # use custom dhcp interface name
+  # Configure sync features; by default all features are enabled.
```

Comment:
If it’s by default true, let’s uncommon those , so CI runs with those enabled

Reply:
there not, comment out by upstream just a copy pasta -> https://github.com/bakito/adguardhome-sync